NPC/corner-turret/Makefile

```diff
@@ -0,0 +1,63 @@
+# Copyright 2015-2019 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+COLOR ?= always # Valid COLOR options: {always, auto, never}
+CARGO = cargo --color $(COLOR)
+TARGET = target/wasm32-unknown-unknown
+DEBUG = $(TARGET)/debug
+RELEASE = $(TARGET)/release
+KEYDIR ?= .keys
+
+.PHONY: all bench build check clean doc test update keys keys-account keys-module
+
+all: build
+
+bench:
+	@$(CARGO) bench
+
+build:
+	@$(CARGO) build
+	wascap sign $(DEBUG)/corner_turret.wasm $(DEBUG)/corner_turret_signed.wasm -i $(KEYDIR)/account.nk -u $(KEYDIR)/module.nk -g -t npc -t avatar-turret2 -n "Turret"
+
+check:
+	@$(CARGO) check
+
+clean:
+	@$(CARGO) clean
+
+doc:
+	@$(CARGO) doc
+
+test: build
+	@$(CARGO) test
+
+update:
+	@$(CARGO) update
+
+release:
+	@$(CARGO) build --release
```

Comment:
Mech actors need to only be signed with the wasmdome:engine capability, and not with wascc:messaging

Reply:
Not sure how we're able to test this actor if it isn't signed with the wasmdome engine provider capability